app/oxalis/nml/NMLService.scala

```diff
@@ -95,13 +95,13 @@ trait NMLParsingService extends LazyLogging {
     var otherFiles = Map.empty[String, TemporaryFile]
     var parseResults = List.empty[NMLParseResult]
     ZipIO.withUnziped(file, includeHiddenFiles = false) { (filename, file) =>
-      if (filename.endsWith(".nml")) {
-        val result = extractFromNML(file, filename)
+      if (filename.toString.endsWith(".nml")) {
```

Comment:
I changes the signatures from `String` to `Path`, since I think it's semantically more fitting. But while `path.endsWith` exists, it checks the whole path segment (i.e. filename) for equality and thus does not work with file extensions